### ISSUE TITLE
Better Error Handler

### ISSUE THREAD

**Issue by vinkla** [gomoob/php-pushwoosh#25]
Today our premium account had ended and we had no idea what was going on until I tried making a request directly against the Pushwoosh API. The answer from the Pushwoosh API was:

```
{#363
  +"status_code": 210
  +"status_message": "This feature is available for Premium accounts only"
  +"response": null
}
```

Though, this is what I got back from this library:

```
ErrorException array_key_exists() expects parameter 2 to be array, null given
    vendor/gomoob/php-pushwoosh/src/main/php/Gomoob/Pushwoosh/Model/Response/CreateMessageResponse.php:43
```
Maybe we can return the error messages whenever the Pushwoosh API return 210?

**Comment by bgaillard** [comment]
Hi @vinkla, this error should have been fixed with https://github.com/gomoob/php-pushwoosh/issues/13. 

Its strange to encounter this problem again, in your response you provided the JSON response, what is the HTTP response code associated to the request ? Is it a 4XX or 5XX response code ? 

If it is I think your problem is related to the following issues : 
* https://github.com/gomoob/php-pushwoosh/issues/24
* https://github.com/gomoob/php-pushwoosh/issues/14

We'll improve error management in version `1.0.6` to fixe those problems.

**Comment by vinkla** [comment]
I can't really remember the response and it is now quite hard to test since my account now is paid. I also think that could be related to #24

**Comment by bgaillard** [comment]
Ok, I think the problem you describe is fixed in version `1.0.5`, but we made an error while publishing this version because its not on packagist https://packagist.org/packages/gomoob/php-pushwoosh. 

So I suppose you were using version `1.0.4` which does not include the fix, its related to https://github.com/gomoob/php-pushwoosh/issues/19. 

I'll publish version `1.0.6` today and I think it will fix this problem definitly.

**Comment by vinkla** [comment]
Aah, that is probably why. Good that we found it then.

**Comment by vinkla** [comment]
Have you setup auto publishing for this  repo on Packagist? Or do you do it manually?

**Comment by bgaillard** [comment]
Yes, I think the problem was that the version was not equal to the GIT tag in the `composer.json` file. 

We've just published a new `1.0.6` version which should fix those problems.
